src/main/lrsql/ops/query/admin.clj

```diff
@@ -61,3 +63,42 @@
           {:account-id account-id
            :username username})
         (bp/-query-all-admin-accounts bk tx)))
+
+(s/fdef query-status
+  :args (s/cat :bk ss/admin-status-backend?
+               :tx transaction?
+               :input ss/query-status-input-spec)
+  :ret ss/query-status-ret-spec)
+
+(defn query-status
+  "Get status information about the LRS including statement counts and other
+  metric information."
+  [bk tx {:keys [include] :as params}]
+  (cond-> {}
+    (include "statement-count")
+    (assoc :statement-count
+           (:scount (bp/-query-statement-count bk tx)))
+
+    (include "actor-count")
+    (assoc :actor-count
+           (:acount (bp/-query-actor-count bk tx)))
+
+    (include "last-statement-stored")
+    (assoc :last-statement-stored
+           (:lstored (bp/-query-last-statement-stored bk tx)))
+
+    (include "platform-frequency")
+    (assoc :platform-frequency
+           (reduce
+            (fn [m {:keys [platform scount]}]
+              (assoc m platform scount))
+            {}
+            (bp/-query-platform-frequency bk tx)))
+
+    (include "timeline")
+    (assoc :timeline
+           (mapv
+            (fn [{:keys [stored scount]}]
+              {:stored (u/pad-time-str stored)
```

Comment:
Is there a reason why we need to pad the timestamp string here?

Reply:
Yes. Since timestamps are ISO zulu (and normalized) in the DB, the timeline query uses a substring call to bucket them. This results in a partial timestamp which needs to have the rest (zeroed) re-added.

Reply:
Hmm so all the timestamp values used in the lrsql app _are_ already normalized, and we'd just have to convert queried timestamps to it?

Reply:
Not sure exactly what you mean, but yeah I think so? Basically what you'd get back for instance for the `hour` unit would be `2023-02-22T12` and this would make it `2023-02-22T12:00:00.000000000Z` so it is valid and normalized.

Reply:
I basically mean timestamp strings that are already used in the app layer, though it seems that they're generally used as Instants and not strings. That said there are other places where timestamp strings are returned, but _they_ don't get padded out, so I'd imagine this padding is required by the UI.

Reply:
Places where timestamp strings are returned, but without calling `pad-time-str` first: https://github.com/yetanalytics/lrsql/blob/d53cb5ce53d352730694a57db8abf8cdcdc3ae9b/src/main/lrsql/ops/query/document.clj#L42

Reply:
As discussed during standup, I'd add a comment where `pad-time-str` is used to clarify why it's used here and not in other places where timestamp strings are returned.

Reply:
https://github.com/yetanalytics/lrsql/pull/282/commits/f410659efe8b3c568e275e0ef419ff29be0e8849